tests/filecheck/dialects/riscv/riscv_ops.mlir

```diff
@@ -6,4 +6,6 @@
   // CHECK: %2 = "riscv.add"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
   %3 = "riscv.sub"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
   // CHECK-NEXT: %3 = "riscv.sub"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+  %4 = "riscv.li"() {"immediate" = 1 : i32}: () -> !riscv.reg<>
```

Comment:
maybe to decouple the order of things, we could change the ssa value names back? so here would be something like this:
```suggestion
  %li = "riscv.li"() {"immediate" = 1 : i32}: () -> !riscv.reg<>
```